pkg/apis/pingcap/v1alpha1/tidbdashboard_types.go

```diff
@@ -107,6 +107,17 @@ type TidbDashboardSpec struct {
 
 	// PreferIPv6 indicates whether to prefer IPv6 addresses for all components.
 	PreferIPv6 bool `json:"preferIPv6,omitempty"`
+
+	//ListenOnLocalhostOnly whether to expose dashboard to 0.0.0.0 or limit it to localhost only
+	// which means it will be accessible only via port-forwarding
+	// Optional: Defaults to false
+	// +optional
+	ListenOnLocalhostOnly *bool `json:"listenOnLocalhostOnly,omitempty" default:"false"`
+
+	// KeyVisualizer is whether to enable Key Visualizer.
+	// Optional: Defaults to true
+	// +optional
+	KeyVisualizer *bool `json:"keyVisualizer,omitempty" default:"true"`
```

Comment:
How about changing it to `DisableKeyVisualizer` with the `false` default?

Reply:
@csuzhangxc, done